package.json

```diff
@@ -19,6 +19,7 @@
     "clean": "tsc --build --clean && tsc --build tsconfig.build.json --clean && rimraf temp .out && lerna run clean",
     "clean:remove-modules": "yarn clean && rimraf node_modules",
     "prettier:all": "prettier --write --config ./prettier.config.js \"**/*\"",
+    "lint:all": "eslint .",
```

Comment:
Not sure if we nee a dedicated script, I would rather vote for changing the current `lint` script to lint everything